Mathlib/LinearAlgebra/FiniteDimensional.lean

```diff
@@ -838,6 +838,8 @@ section DivisionRing
 variable [DivisionRing K] [AddCommGroup V] [Module K V] {V₂ : Type v'} [AddCommGroup V₂]
   [Module K V₂]
 
+/-- If a submodule is less than or equal to a finite-dimensional
```

Comment:
This is the natural language that corresponds to the Lean code:
```suggestion
/-- If a submodule is contained in a finite-dimensional
```
(same below: 3 occurrences total)